pkg/health-check/cmd.go

```diff
@@ -0,0 +1,613 @@
+package health_check
```

Comment:
along with the package name change can we update the file to be post_analysis.go instead of cmd?